cfme/tests/networks/nuage/test_inventory.py

```diff
@@ -180,3 +181,18 @@ def test_network_port_l2_container(setup_provider_modscope, provider, with_nuage
         ('relationships', 'Cloud tenant'): tenant_name,
         ('relationships', 'Cloud subnets'): '1',
     })
+
+
+def test_router_add_subnet(setup_provider_modscope, provider, with_nuage_sandbox_modscope):
```

Comment:
Please move this test into a new nuage/operations.py file. Also, please use the regular `with_nuage_sandbox` fixture here, not the modscope one because we're adding subnet to it and it's better to discard it afterwards.